src/components/Community.astro

```diff
@@ -0,0 +1,77 @@
+---
+import { Image } from "astro:assets";
+import Coin2_orange_shiny from "../assets/Coin2_orange_shiny.svg";
+---
+
+<!-- -->
+<div class="mx-auto max-w-xl text-center">
+  <div class="-space-y-2">
+    <div class="space-y-2">
+      <p class="font-extrabold text-white text-2xl">Come to DeltaHacks X to</p>
+      <h1
+        class="text-6xl font-black font-display text-fill-transparent text-stroke-white text-stroke-2"
+      >
+        JOIN A GROWING
+      </h1>
+    </div>
+    <div class="space-y-3">
+      <h1 class="text-7xl font-bold text-yellow-400">COMMUNITY</h1>
+    </div>
+  </div>
+</div>
+<div class="mx-auto text-center">
+  <div class="grid grid-rows-1 grid-flow-col p-8">
+    <div class="relative w-80 -rotate-12 translate-x-52 -translate-y-10 p-8">
```

Comment:
Again, avoid using translate, use individual padding instead for each item
eg use a top padding equivalent to 10px instead of translating it down by 10